### ISSUE TITLE
Adding a cshtml file to a Blazor project results in multiple item groups getting added to the project file

### ISSUE THREAD

**Issue by danroth27** [aspnet/Blazor#316]
VSIX version: 15.6.10124

Repro steps:
- Create a new Blazor project
- Add a new Razor View to the Pages folder using the VS item template
- View the csproj file

Expected result:
- No change to the csproj

Actual result:

```xml
<Project Sdk="Microsoft.NET.Sdk.Web;Microsoft.NET.Sdk.Razor/2.1.0-preview2-30230">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <RunCommand>dotnet</RunCommand>
    <RunArguments>blazor serve</RunArguments>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.0-preview2-30230" PrivateAssets="all" />
    <PackageReference Include="Microsoft.AspNetCore.Blazor.Browser" Version="0.1.0-rc1-10117" PrivateAssets="all" />
    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="0.1.0-rc1-10117" />
    <DotNetCliToolReference Include="Microsoft.AspNetCore.Blazor.Cli" Version="0.1.0-rc1-10117" />
  </ItemGroup>

  <ItemGroup>
    <BlazorGenerate Remove="Pages\Todo.cshtml" />
  </ItemGroup>

  <ItemGroup>
    <Content Remove="Pages\Todo.cshtml" />
  </ItemGroup>

  <ItemGroup>
    <None Include="Pages\Todo.cshtml" />
  </ItemGroup>

</Project>
```

**Comment by rynowak** [comment]
Yeah, it's not good if this is happening because it means that the item is going to be excluded from compilation.

**Comment by rynowak** [comment]
Is it possible in this case that you renamed the file? I've seen this kind of thing happen when you rename a file to have some other file extension and then back to `.cshtml`

**Comment by danroth27** [comment]
Not in this particular case. The repro steps are exactly as listed above.

**Comment by danroth27** [comment]
I do see other oddities in the csproj file though when I copy an existing cshtml file within the same project and then rename it.

**Comment by danroth27** [comment]
@SteveSandersonMS  Can we get someone to take a look and figure out what is going on here?

**Comment by MisinformedDNA** [comment]
> I do see other oddities in the csproj file though when I copy an existing cshtml file within the same project and then rename it.

Seeing this as well.

**Comment by rynowak** [comment]
OK thanks for the bump on this. I know it's annoying 😢.

**Comment by rynowak** [comment]
This should be fixed now in dev, and will be part of 0.4
